workers/utils.py

```diff
@@ -341,7 +342,6 @@ class PausedFuzzers:
     active.
     """
 
-    FUZZER_CMD_PORT = 7055
     SERVICE = 'nayduck-fuzzer.service'
```

Comment:
Should we put that in utils as well?

Reply:
Not sure what you mean? I just moved that up to the top of workers.utils, so that fuzzers.main could access it

Reply:
Sorry, I meant top level.  But now that I think of it I guess it’s not as weird that SERVICE is not together with the port number.